src/openloco/windows/build_vehicle.cpp

```diff
@@ -120,11 +120,11 @@ namespace openloco::ui::build_vehicle
     static window_event_list _events;
 
     static void sub_4B92A5(ui::window* window);
-    static void sub_4C28D2(ui::window* window);
-    static void sub_4C2D8A(ui::window* window);
-    static void sub_4C1CBE(ui::window* window);
-    static void sub_4A3A06(uint8_t track_type, bool is_road);
-    static void sub_4C2865(ui::window* window);
+    static void SetDisabledTransportTabs(ui::window* window);
+    static void SetTrackTypeTabs(ui::window* window);
+    static void ResetTrackTypeTabSelection(ui::window* window);
+    static void SetTopToolbarLastTrack(uint8_t track_type, bool is_road);
+    static void SetTransportTypeTabs(ui::window* window);
```

Comment:
I really appreciate you naming these functions. However, I would like to continue to use camelCase for function names to distinguish them from types (which should use TitleCase).